WPMedia/fusion-news-theme-blocks
# Themes 1222 - Logical Properties on Article Body Block

## Description

This PR updates the logical properties on the article-body-block.

## Jira Ticket

- [THEMES-1222](https://arcpublishing.atlassian.net/browse/THEMES-1222)

## Acceptance Criteria

1. The above logical properties are applied and tested on the article-body-block.
2. Storybook is updated with RTL. 

## Test Steps

1. Checkout this branch `git checkout THEMES-1222`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/article-body-block`
3. Go to http://localhost/pf/2022/09/15/the-kitchen-sink-of-composer-features/?_website=the-gazette
4. Verify it matches the designs in Figma.
5. Set `"textDirection": "rtl"` in the `"siteProperties"` object of the `blocks.json`.
6. Restart PageBuilder and verify that the styling adapts to the new text direction.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._


[THEMES-1222]: https://arcpublishing.atlassian.net/browse/THEMES-1222?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
